sympy/printing/mathematica.py

```diff
@@ -101,6 +102,8 @@ def _print_Function(self, expr):
                     return "%s[%s]" % (mfunc, self.stringify(expr.args, ", "))
         return expr.func.__name__ + "[%s]" % self.stringify(expr.args, ", ")
 
+    _print_Expr = _print_Function
```

Comment:
Would it suffice to write  `_print_MinMaxBase = _print_Function`?

Reply:
I just tested and that works too.

Reply:
I think that making the change for MinMaxBase only would be preferable. I don't know the syntax of Mathematica but one should probably avoid using the function syntax for all expressions (that do not have another printing method defined).

Reply:
OK, that makes sense. I was just copying from codeprinter.py which defines _print_Expr = _print_Function. Pushed a new commit to change it.